src/style/footer.module.css

```diff
@@ -0,0 +1,20 @@
+.footer {
```

Comment:
nit: the line spacing in `funding` section looks larger than in intro/outro of `acknowledgments` by eye. Could this be `HTMLContent` renders funding text differently?